src/powershell/install-boxstarter.ps1

```diff
@@ -0,0 +1,42 @@
+# Install and configure Boxstarter
+# See: https://boxstarter.org/
+function Install-Boxstarter {
+    Invoke-Expression ((New-Object System.Net.WebClient).DownloadString("https://boxstarter.org/bootstrapper.ps1"))
+    Get-Boxstarter -Force
+}
+
+function Set-BoxstarterConfiguration {
+    $Boxstarter.RebootOk = (-not $noReboots.IsPresent)
+    $Boxstarter.AutoLogin = $true
+    $Boxstarter.SuppressLogging = $true
+    $global:VerbosePreference = "SilentlyContinue"
+
+    # Add Chocolatey and Mandiant VM-Packages repositories as Boxstarter sources
+    Set-BoxstarterConfig -NugetSources "$desktopPath;.;https://www.myget.org/F/vm-packages/api/v2;https://chocolatey.org/api/v2"
+
+    # Import Boxstarter Chocolatey module to enable Chocolatey commands
+    Import-Module "${Env:ProgramData}\boxstarter\boxstarter.chocolatey\boxstarter.chocolatey.psd1" -Force
+}
+
+# Change some Windows Explorer configuration options. This matches the configuration
+# performed for Commando VM images:
+# https://github.com/mandiant/commando-vm/blob/688599b87966c757524a7a19a9a6de2359885b3f/install.ps1#L1050
+function Configure-Windows-Explorer {
```

Comment:
This is entirely my fault. I noticed that the highlighting didn't treat this as a function so I looked up the [function naming conventions](https://learn.microsoft.com/en-us/powershell/scripting/learn/ps101/09-functions?view=powershell-7.3). Please feel free to group this with the related suggestion into one commit (you can do that from the `Files changed` tab).

```suggestion
function Enable-WindowsExplorerOptions {
```